lib/src/viam_sdk.dart

```diff
@@ -1,29 +1,47 @@
 import 'package:auth0_flutter/auth0_flutter.dart';
 import 'package:grpc/grpc_connection_interface.dart';
-import 'package:viam_sdk/src/domain/app/service/app_api_data_source.dart';
-import 'package:viam_sdk/src/domain/camera/service/camera_api_service.dart';
-import 'package:viam_sdk/src/domain/data/service/data_api_service.dart';
-import 'package:viam_sdk/src/domain/movement/service/viam_movement_service.dart';
-import 'package:viam_sdk/src/domain/resource/service/viam_resource_service.dart';
-import 'package:viam_sdk/src/domain/sensor/service/viam_sensor_service.dart';
-import 'package:viam_sdk/src/viam_sdk_impl.dart';
+
+import './app/app.dart';
+import './app/data.dart';
+import './domain/app/service/app_api_data_source.dart';
+import './domain/camera/service/camera_api_service.dart';
+import './domain/data/service/data_api_service.dart';
+import './domain/movement/service/viam_movement_service.dart';
+import './domain/resource/service/viam_resource_service.dart';
+import './domain/sensor/service/viam_sensor_service.dart';
+import './gen/app/v1/app.pbgrpc.dart';
+import './robot/client.dart';
+import './viam_sdk_impl.dart';
 
 abstract class Viam {
+  static Viam withAccessToken(String accessToken) {
+    return ViamImpl.withAccessToken(accessToken);
+  }
+
+  AppClient get appClient;
+  DataClient get dataClient;
+
+  Future<RobotClient> getRobotClient(Robot robot);
+
+  @Deprecated('This function will be removed prior to beta launch')
   static Viam instance() => ViamImpl();
```

Comment:
do we want to give access to the object without an access token constructor?

Reply:
Yes -- because DoR might still be using it. I'm not sure where they are in their migration so I don't want to remove it (see above not about private constructor for `ViamImpl`)